packages/client/src/pages/sign-in.tsx

```diff
@@ -53,38 +61,46 @@ export const SignIn = () => {
         }}
       >
         <ProjectDisplay project={project} />
-        <Formik
-          validateOnBlur={false}
-          validateOnChange={false}
-          validationSchema={LoginValidation}
-          initialValues={{ email: '', password: '' }}
-          onSubmit={async ({ email, password }) => {
-            await loginEmail({ variables: { email, password, projectId: project?.id || '' } });
-          }}
-        >
-          <Form>
-            <Card>
-              <CardHeader title="Sign in" />
-              <CardContent>
-                <TextInput autoFocus fullWidth name="email" label="Email Address" type="email" autoComplete="email" margin="normal" required />
-                <PasswordInput name="password" label="Password" fullWidth autoComplete="current-password" required margin="normal" />
-              </CardContent>
-            </Card>
-            <SubmitButton fullWidth variant="contained" color="primary" sx={{ my: 2 }}>
-              Sign In
-            </SubmitButton>
-          </Form>
-        </Formik>
+        {project?.authMethods.emailAuth ? (
```

Comment:
If you are conditionally rendering something where on condition renders null, use can just use the && operator
```suggestion
        {project?.authMethods.emailAuth && (
```
And then just close it
```
)}
```